### PR TITLE
Add parameters to ceph integration

### PR DESCRIPTION
**Pull request by stamak**
Closes: #321

### CODE REVIEW

File: manifests/integrations/ceph.pp [hunk 1]
```diff
@@ -3,16 +3,24 @@
...
+#   $ceph_cmd
+#       Optional ceph cmd
 
 # Sample Usage:
 #
 #  class { 'datadog_agent::integrations::ceph' :
 #  }
 #
 class datadog_agent::integrations::ceph(
+  $tags = [ 'name:ceph_cluster' ],
```
Comment (truthbk) [inline manifests/integrations/ceph.pp]: Ugh... I kind of hate that we have to do it this way, but it is indeed the only way to keep backward compatibility 👍

File: templates/agent-conf.d/ceph.yaml.erb [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+<% if @tags and ! @tags.empty? -%>
   - tags:
-    - name:ceph_cluster
+  <%- @tags.each do |tag| -%>
+    - <%= tag %>
+  <%- end -%>
+<% end -%>
 
-    ceph_cmd: /usr/bin/ceph
+    ceph_cmd: <%= @crm_mon_cmd %>
```
Comment (truthbk) [inline templates/agent-conf.d/ceph.yaml.erb]: I'll have to take a better look, but is this right `@crm_mon_cmd`?

**Comment by truthbk** [comment]
Hi @stamak!

Thanks a lot for your work! Would it be at all possible for you to write a couple spec tests to cover the new usage scenarios? It would help us keep track for future breakage.

Also, there was an issue breaking the CI. Could you please rebase to the latest `master`? After bringing in the latest stuff to your fork:

```
git checkout master
git pull
git checkout ceph_params
git rebase master
git push -f origin ceph_params
```

That should do it. Thank you again for your work and time 🙇

File: templates/agent-conf.d/ceph.yaml.erb (hunk 2 above)
Comment (stamak) [inline templates/agent-conf.d/ceph.yaml.erb]: it's mistake, thanks

**Comment by stamak** [comment]
@truthbk done ;)

**Comment by truthbk** [comment]
Thank you @stamak, this looks good! Merging it in! 🙇

**Pull request merged**